intl.emu

```diff
@@ -230,7 +220,7 @@
           1. Else,
             1. Set _n_ to _n2_.
             1. Let _xFinal_ be _r2_.
-        1. If _n_ = 0, let _m_ be *"0"*. Otherwise, let _m_ be the String consisting of the digits of the decimal representation of _n_ (in order, with no leading zeroes).
+        1. If _n_ = 0 <ins>and _m_ is *undefined*</ins>, <del>let</del><ins>set</ins> _m_ <del>be</del><ins>to</ins> *"0"*. Otherwise, <del>let</del><ins>set</ins> _m_ <del>be</del><ins>to</ins> the String consisting of the digits of the decimal representation of _n_ (in order, with no leading zeroes).
```

Comment:
Instead of checking if _m_ is undefined, you can indent this inside the Else block.

Reply:
done